### DISCUSSION TITLE
Reusable loading Bar Component in Blazor

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#7] [General]
# Reusable loading Bar Component in Blazor

In our Blazor components we often call a repository or web API which takes some time. So it is nice to indicate to an user what the current state is. That is why we will create a small indicator even with steps to indicate how far we are in the progress.

https://steven-giesel.com/blogPost/d1884385-90de-43a1-ae24-0178495f1484

**Comment by AliCharper** [comment]
Hi,

It seems there is a bug even in your sample

 Unhandled exception rendering component: Object of type 'Loading.Shared.LoadingComponent' does not have a property matching the name 'ChildContent'.

**Reply by linkdotnet** [reply]
Thanks for noticing. Indeed a named the varialbe wrong (`ChildComponent`) instead of `ChildContent`. Big thank you @AliCharper 

I fixed it in the blog post as well as the sample code.
